indico/modules/events/surveys/controllers/management/results.py

```diff
@@ -62,7 +62,7 @@ def _process(self):
         submission_ids = set(map(int, request.form.getlist('submission_ids')))
         headers, rows = generate_spreadsheet_from_survey(self.survey, submission_ids)
         filename = 'submissions-{}.{}'.format(self.survey.id, self.EXT)
-        return self.FUNC(filename, headers, rows)
+        return self.FUNC(filename, headers, rows, tz=self.event_new.tzinfo)
```

Comment:
but when we generate the data to pass to the send_csv, we are not passing datetime objects, so even if we pass the timezone, nothing changes.